src/apps/activities/domain/conditions.py

```diff
@@ -133,39 +637,67 @@ class ScoreBoolCondition(BaseCondition):
     | NotEqualCondition
     | BetweenCondition
     | OutsideOfCondition
+    | GreaterThanDateCondition
+    | BetweenTimeRangeCondition
+    | LessThanDateCondition
+    | EqualToDateCondition
+    | BetweenDatesCondition
+    | NotEqualToDateCondition
+    | OutsideOfDatesCondition
+    | GreaterThanTimeRangeCondition
+    | LessThanTimeRangeCondition
+    | EqualToTimeRangeCondition
+    | NotEqualToTimeRangeCondition
+    | OutsideOfTimeRangeCondition
+    | EqualToRowOptionCondition
+    | NotEqualToRowOptionCondition
+    | IncludesOptionPerRowCondition
+    | NotIncludesOptionPerRowCondition
+    | GreaterThanSliderRowCondition
+    | LessThanSliderRowCondition
+    | EqualToSliderRowCondition
+    | NotEqualToSliderRowCondition
+    | BetweenSliderRowCondition
+    | OutsideOfSliderRowCondition
+    | GreaterThanTimeCondition
+    | LessThanTimeCondition
+    | EqualToTimeCondition
+    | NotEqualToTimeCondition
+    | BetweenTimeCondition
+    | OutsideOfTimeCondition
 )
 
 ScoreCondition = (
-    GreaterThanCondition
-    | LessThanCondition
-    | EqualCondition
-    | NotEqualCondition
-    | BetweenCondition
-    | OutsideOfCondition
+    ScoreGraterThanCondition
+    | ScoreLessThanCondition
+    | ScoreEqualCondition
+    | ScoreNotEqualCondition
+    | ScoreBetweenCondition
+    | ScoreOutsideOfCondition
 )
 SectionCondition = (
-    IncludesOptionCondition
-    | NotIncludesOptionCondition
-    | EqualToOptionCondition
-    | NotEqualToOptionCondition
-    | GreaterThanCondition
-    | LessThanCondition
-    | EqualCondition
-    | NotEqualCondition
-    | BetweenCondition
-    | OutsideOfCondition
+    ScoreIncludesOptionCondition
+    | ScoreNotIncludesOptionCondition
+    | ScoreEqualToOptionCondition
+    | ScoreNotEqualToOptionCondition
+    | ScoreGraterThanCondition
+    | ScoreGraterThanCondition
+    | ScoreEqualCondition
+    | ScoreNotEqualCondition
+    | ScoreBetweenCondition
+    | ScoreOutsideOfCondition
     | ScoreBoolCondition
 )
 AnyCondition = (
-    IncludesOptionCondition
-    | NotIncludesOptionCondition
-    | EqualToOptionCondition
-    | NotEqualToOptionCondition
-    | GreaterThanCondition
-    | LessThanCondition
-    | EqualCondition
-    | NotEqualCondition
-    | BetweenCondition
-    | OutsideOfCondition
+    ScoreIncludesOptionCondition
+    | ScoreNotIncludesOptionCondition
+    | ScoreEqualToOptionCondition
+    | ScoreNotEqualToOptionCondition
+    | ScoreGraterThanCondition
+    | ScoreGraterThanCondition
+    | ScoreEqualCondition
+    | ScoreNotEqualCondition
+    | ScoreBetweenCondition
+    | ScoreOutsideOfCondition
     | ScoreBoolCondition
```

Comment:
Why did you rename these from their generic form example `GreaterThanCondition` to `Score-` prefix? I think it was better with the original naming.

Additionally, there's a typo in "Greater" (Grater)